WebApiCoreSeed.WebApi/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.WebApi.Middleware
+{
+    /// <summary>
+    /// Middleware to handle global authorization policies
+    /// </summary>
+    public class AuthorizationMiddleware
```

Comment:
Guys, I vouch for keeping the example and add the comments as Marce suggested. This way we can either remove it or use it if needed. Please let me know if everybody agrees. Thanks